net/instaweb/rewriter/rewrite_query.cc

```diff
@@ -545,7 +545,7 @@ RewriteQuery::Status RewriteQuery::ScanNameValue(
       if (pairs[i] == "no-transform") {
         // TODO(jmarantz): A .pagespeed resource should return un-optimized
         // content with "Cache-Control: no-transform".
-        options->set_enabled(RewriteOptions::kEnabledOff);
+        options->set_enabled(RewriteOptions::kEnabledStandby);
```

Comment:
We do accept `?PageSpeed=off` on resources; that's one way to see what a resource would look like without IPRO.

But I'm still not clear on what you're objecting to here.  Would it help to talk in person?